build/portal/html/admin/index.php

```diff
@@ -375,6 +375,41 @@ function formatOffset($offset) {
                     </div>
                 </div>
                 <div role="tabpanel" class="tab-pane fade" id="navigation">
+                  <div class="panel panel-default">
+                      <div class="panel-heading">API Key</div>
+                      <div class="panel-body">
+                        <div class="form-group">
+                                <label for="BingMapsAPI">Bing Maps API Key</label>
+                                <input type="text" class="form-control" id="bingMapAPIKey" name="bingMapAPIKey"
+                                        value="<?php  if(!empty($_POST['bingMapAPIKey'])){
+                                        $apikey = $_POST['bingMapAPIKey'];
+                                        $path_to_file = '/usr/share/dump1090-mutability/html/config.js';
+                                        $file_contents = file_get_contents($path_to_file);
+                                        $file_contents = preg_replace('/BingMapsAPIKey = ([a-zA-Z0-9"]+)/', "BingMapsAPIKey = '" . $_POST['bingMapAPIKey'] . "'" , $file_contents);
+                                        file_put_contents($path_to_file,$file_contents);
+                                        }
+                                        else {
+                                          echo file_get_contents('/usr/share/dump1090-mutability/html/config.js');

```

Comment:
same case with the mapzen API Key below. It would also be a good idea:

1. Not to hardcode the JS file's absolute path (maybe a relative path would be better)
2. As the logic for saving and retrieving the API keys is repeated for each API key, it would make sense to make a function.